### ISSUE TITLE
Inconsistencies / typos in test comments and function names.

### ISSUE THREAD

**Issue by christophherr** [Getbeans/Beans#74]
Probably just a few copy & paste leftovers.
I put my suggestions in a diff, I thought that would be easier than listing things here.
https://github.com/christophherr/Beans/commit/2cce8ee20fbdf65b06b6ac03688112924e616f45

**Comment by hellofromtonya** [comment]
Once PR #75 is merged, then go ahead and create a new branch for the changes you listed.  Then submit a new PR.  Let me know if you have any questions.

**Comment by hellofromtonya** [comment]
@christophherr PR #75 has now been merged into the `development` branch.  Go ahead and update your local repo via:

```
git checkout development
git fetch upstream
git merge upstream/development
git push origin development
```

Then go ahead with the fixes that you noted above on a separate branch. Make sure that you run:

```
composer phpcs-tests
```

and then

```
composer test-unit
```

When all passes, push your branch and open a PR.
